src/config.ts

```diff
@@ -52,11 +54,15 @@ export const DEFAULT_CONFIG: SWACLIConfig = {
   devserverTimeout: parseInt(SWA_CLI_SERVER_TIMEOUT || "60", 10),
   open: useEnvVarOrUseDefault(SWA_CLI_OPEN_BROWSER, false),
   githubActionWorkflowLocation: SWA_RUNTIME_WORKFLOW_LOCATION ? SWA_RUNTIME_WORKFLOW_LOCATION : undefined,
+  appDevserverUrl: SWA_CLI_APP_DEVSERVER_URL || undefined,
+  apiDevserverUrl: SWA_CLI_API_DEVSERVER_URL || undefined,
+
+  // swa deploy options
   env: SWA_CLI_DEPLOY_ENV || "preview",
   appName: SWA_CLI_APP_NAME || undefined,
   dryRun: useEnvVarOrUseDefault(SWA_CLI_DEPLOY_DRY_RUN, false),
-  appDevserverUrl: SWA_CLI_APP_DEVSERVER_URL || undefined,
-  apiDevserverUrl: SWA_CLI_API_DEVSERVER_URL || undefined,
+  apiLanguage: SWA_CLI_API_LANGUAGE || "node",
+  apiVersion: SWA_CLI_API_VERSION || "16",
```

Comment:
I think these values should be set during swa init after we detect the user's config. 
If the user is using a dotnet API and forgot to provide these values during deployment, it's gonna fail!

Reply:
@manekinekko do we have some tests for testing all kinds of apps after any new changes are pushed?

Reply:
@manekinekko how about during the swa init, we prompt the user to initialize these values along with App-location and api-location. What do you think about this?

Reply:
@sinedied I've added the prompts in the init file, should I remove the code changes in the deploy.ts?

Reply:
Which changes?
I think the default values should be set in the deploy command parameters (like other params) and not hardcoded

Reply:
I agree with @sinedied we should not hardcode these values in the config and should handle the `undefined` case in the deploy.

```suggestion
  apiLanguage: SWA_CLI_API_LANGUAGE || undefined,
  apiVersion: SWA_CLI_API_VERSION || undefined,
```